pydanticrud/backends/dynamodb.py

```diff
@@ -217,16 +212,16 @@ def __init__(self, cls, result, serialized_items):
 
 class Backend:
     def __init__(self, cls):
-        cfg = cls.model_config
+        cfg = cls.DBConfig
```

Comment:
```suggestion
        cfg = cls.db_config
```
to match the snake case of everything else.

Reply:
Done